src/main/java/com/spotinst/sdkjava/model/K8sVngClient.java

```diff
@@ -0,0 +1,133 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.exception.HttpError;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.bl.ocean.kubernetes.*;
+import com.spotinst.sdkjava.model.requests.ocean.kubernetes.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class K8sVngClient {
+    private static final Logger     LOGGER = LoggerFactory.getLogger(K8sVngClient.class);
+
+    private String                  authToken;
+    private String                  account;
+    private ISpotK8sVngRepo   		K8sVngRepo;
+
+    public ISpotK8sVngRepo getK8sVngRepo() {
+        return K8sVngRepo;
+    }
+
+    public K8sVngClient(String authToken, String account) {
+        this.authToken = authToken;
+        this.account = account;
+
```

Comment:
we can set K8sVngRepo in constructor, no need to call with set method.
So Add this.K8sVngRepo = SpotinstRepoManager.getInstance().getK8sVngRepo(); here. 
And remove setMethod. 

reference: SpotOceanEcsClusterClient